CMakeLists.txt

```diff
@@ -122,28 +122,34 @@ set_local_or_system_option(
     "GTEST" OFF
     "Build the GoogleTest framework by providing your own copy of the repo in ext/googletest (see Local Builds in README)"
 )
+
```

Comment:
Does this pull request also fix or partially address https://github.com/AOMediaCodec/libavif/issues/2277? I am wondering if this pull request replaces https://github.com/AOMediaCodec/libavif/pull/2290.

Reply:
Almost but not quite: the bug still happens with #2290 2290

Reply:
By "the bug still happens with #2290", were you referring to https://github.com/AOMediaCodec/libavif/issues/2277 or https://github.com/AOMediaCodec/libavif/issues/2300?

I was asking about https://github.com/AOMediaCodec/libavif/issues/2277.

Reply:
With the rebase, this PR only fixes #2300